inbm/fpm/inb-provision-certs/main.go

```diff
@@ -57,9 +58,18 @@ func main() {
 	setUpMqttCaDirectories(secretDir, publicDir, daysExpiry)
 	setUpMqttBrokerDirectories(secretDir, publicDir, daysExpiry)
 
-	for _, each := range []string{
+	agents := []string{
 		"dispatcher-agent", "telemetry-agent", "diagnostic-agent", "configuration-agent",
-		"cloudadapter-agent", "inbc-program", "vision-agent", "node-agent", "cmd-program"} {
+		"cloudadapter-agent", "inbc-program", "cmd-program"}
+
+	// set up certs also for ucc-native-service, in UCC mode
+	uccFlagPath := "/etc/intel-manageability/public/ucc_flag"
+	if content, err := ioutil.ReadFile(uccFlagPath); err == nil &&
+		strings.TrimSpace(string(content)) == "TRUE" {
+		agents = append(agents, "ucc-native-service")
```

Comment:
Are we generating cert/key pair for TC agents which are not getting installed?

Reply:
@pillaiabhishek good catch. i fixed this and reran ucc integration test. pass.